jenkins_jobs/trigger_nightly_host_os_build.groovy

```diff
@@ -69,9 +68,8 @@ job('trigger_nightly_host_os_build') {
     downstreamParameterized {
       trigger('build_host_os_iso') {
         block {
-          buildStepFailure('FAILURE')
-          failure('FAILURE')
-          unstable('UNSTABLE')
+          buildStepFailure('UNSTABLE')
+          failure('UNSTABLE')
```

Comment:
It seems to me this sets the build as "unstable" even if build fails. If so, this does not make sense. Wouldn't it be better to split ISO build and upload in separate jobs and react differently to a failure in each of them?